### DISCUSSION TITLE
Set AC temperature ?

### DISCUSSION THREAD

**Discussion by user7590** [TobiasLaross/Hass-Lynk-Co#42] [Q&A]
Anyone tried **setting** the AC temperature (or setting it to MAX A/C or so, wich would be different from "climate_level" HIGH, which i think is just the fan setting?)?

**Comment by TobiasLaross** [comment]
Interesting, I just assumed that HIGH and LOW was a bit higher/lower temperature than medium. I have not used the official app but I don’t think that app expose level but using medium it will pre heat the car to 22°C if I remember correctly, so setting other temperatures in the car will likely not help

**Comment by user7590** [comment]
that IS intereseting yes. So you think that setting means the height of the temperature? I presumed it was fanspeed :-)
